apps/web/README.md

```diff
@@ -0,0 +1,47 @@
+# Welcome to Remix + Cloudflare!
```

Comment:
Would be nice to commit a basic version of our own readme here - describe the app, present it to the community at large and describe what we are doing here.

Reply:
have some description in mind?